src/components/v2v3/shared/Allocation/AllocationList.tsx

```diff
@@ -82,7 +83,9 @@ export const AllocationList = ({
       const totalAmount = parseFloat(fromWad(totalAllocationAmount))
       const removedAmount = (allocation.percent / 100) * totalAmount
 
-      const totalAmountAfterRemoval = totalAmount - removedAmount
+      const totalAmountAfterRemoval = roundIfCloseToNextInteger(
```

Comment:
This was a simple enough fix - basically just sometimes left over with a really really small decimal value that causes an underflow